### PR TITLE
Migrate Swedish dev DB on deploy

### PR DESCRIPTION
**Pull request by lorentzlasson**
#530

---

Tested on devices

- [ ] Desktop 💻
- [ ] Mobile 📱

Tests

- [ ] All tests are running ✔️
- [ ] Test are updated 🧪
- [ ] Code Review 👩‍💻
- [ ] QA 👌

⏲️ Time spent on CR:

⏲️ Time spent on QA:

### CODE REVIEW

File: cloud.dev.deploy.yaml [hunk 1]
```diff
@@ -27,8 +27,20 @@ steps:
       - "--allow-unauthenticated"
 
   - name: "gcr.io/cloud-builders/gcloud"
-    id: "deploy-sweden"
+    id: "migrate-db-sweden"
     waitFor: ["build"]
+    args:
+      - "run"
+      - "jobs"
+      - "execute"
```
Comment (lorentzlasson) [inline cloud.dev.deploy.yaml:35]: Going with the execute command now. I couldn't find a way to upsert (create or update) the job nicely, so went with creating the job manually outside CI and then let CI execute it only. It's similar to what we do for Cloud Run servives

File: cloud.dev.deploy.yaml [hunk 2]
```diff
@@ -27,8 +27,20 @@ steps:
       - "--allow-unauthenticated"
 
   - name: "gcr.io/cloud-builders/gcloud"
-    id: "deploy-sweden"
+    id: "migrate-db-sweden"
     waitFor: ["build"]
+    args:
+      - "run"
+      - "jobs"
```
Comment (lorentzlasson) [inline cloud.dev.deploy.yaml:34]: https://cloud.google.com/run/docs/create-jobs

File: cloud.dev.deploy.yaml (hunk 1 above)
Comment (adamalfredsson) [reply cloud.dev.deploy.yaml:35]: What happens should the migration job fail?

Comment (lorentzlasson) [reply cloud.dev.deploy.yaml:35]: This step return an exit code 1 which prevent the next step (deployment) from being run

**Review by adamalfredsson — APPROVED** [review]
Looks good! I'm not sure if it makes sense, but if there's any way to add the job configuration to this repository, I think that would be neat!

**Comment by lorentzlasson** [comment]
> Looks good! I'm not sure if it makes sense, but if there's any way to add the job configuration to this repository, I think that would be neat!

Yeah that's a good idea. As a comment somewhere I guess.

In some IaC thing would have been nice if we had one

**Pull request merged**